src/cmr/concepts/__tests__/concept.test.js

```diff
@@ -47,5 +47,123 @@ describe('collection', () => {
+          const conceptId = 'C1200000000-EDSC'
+          const jsonKeys = ['nestedKey', 'nestedKey2']
+          const jsonResponse = {
+            data:
+                {
+                  hits: 1,
+                  took: 9,
+                  feed: {
+                    entry: [{
+                      concept_id: conceptId,
+                      nested_key: [
+                        {
+                          key1: 'value',
+                          key2: 'value2',
+                          snake_key_1: 'value3',
+                          snake_key_2: 'value4'
+                        },
+                        {
+                          key1: 'value',
+                          key2: 'value2',
+                          snake_key_1: 'value3'
+                        }
+                      ],
+                      nested_key_2: {
+                        url: 'https://mock.html',
+                        format: 'Zarr',
+                        description: 'Other brief end user information can go here.',
+                        mock_distribution_information: {
+                          region: 'us-west-2',
+                          s_3_bucket_and_object_prefix_names: [
+                            'mock-name-1',
+                            'mock-name-2'
+                          ],
+                          s_3_credentials_api_endpoint: 'https://s3mock.html',
+                          s_3_credentials_api_documentation_url: 'https://s3Docsmock.html'
+                        },
+                        mock_chunking_information: 'Chunk size for this example is 1MB. It is optimized for time series.'
+                      }
+                    }]
+                  }
+                }
+          }
+          concept.parseJson(jsonResponse, jsonKeys)
+          expect(concept.items[conceptId].nestedKey).toEqual([
+            {
+              key1: 'value',
+              key2: 'value2',
+              snakeKey1: 'value3',
+              snakeKey2: 'value4'
+            },
+            {
+              key1: 'value',
+              key2: 'value2',
+              snakeKey1: 'value3'
+            }
+          ])
+
+          expect(concept.items[conceptId].nestedKey2).toEqual({
+            description: 'Other brief end user information can go here.',
+            format: 'Zarr',
+            mockChunkingInformation: 'Chunk size for this example is 1MB. It is optimized for time series.',
+            mockDistributionInformation: {
+              region: 'us-west-2',
+              s3BucketAndObjectPrefixNames: ['mock-name-1', 'mock-name-2'],
+              s3CredentialsApiDocumentationUrl: 'https://s3Docsmock.html',
+              s3CredentialsApiEndpoint: 'https://s3mock.html'
+            },
+            url: 'https://mock.html'
+          })
+        })
+      })
+
+      describe('the nested key is tag', () => {
+        test('the child-keys of the tag return snake_cased', () => {
```

Comment:
Change this test name to be more clear about what is happening in the code. The child keys aren't being returned snake_cased, they are being returned in their original casing